index.html

```diff
@@ -75,6 +75,53 @@
         }
       }
 
+      function debug() {
+        console.log("Frontend DEBUG button pressed!");
+        // http://localhost:8538/helloworld/younes
+        console.log(httpRequest("GET", "http://localhost:8538/helloworld/younes", null, null));
+      }
+
+      function httpRequest(verb, url, headers, data) {
```

Comment:
These are included here for debugging purposes only and can be removed once the HTTP request logic is stable